### PR TITLE
fix(charts): Remove to-opaque and make labels for pie chart use the same color as category chart

### LINKED ISSUE #11663: Category chart and pie chart theme inconsistencies
## Description  
Noticed some odd inconsistencies in the category chart and the pie chart themes and want to see if that's intentional or something we can clear out

 * igniteui-angular version: 13.x.x
 * browser: N/A

## Steps to reproduce  

1. Run the sample and inspect the charts
2. Change the theme to `dark-theme` mixin and compare again

## Result  
- Light theme: Category chart sets gray 700 to all axis strokes and labels
- Dark theme: Category Chart sets gray 400 (using apparently old format with `to-opaque: #fff`). Is this intentionally different level of gray from the light theme? Bit odd for one and also less contrast-y.
- Pie Chart: Sets gray 900 for both light and dark theme, making its labels nearly black/white depending on theme and visibly different from the category chart.

There appear to be some variations from the intended designs:
- Category chart sets all labels, title and axis strokes to the same color. From what I can see in designs they are all different shades actually. Guessing @tosharky can clarify.
- The Pie didn't have a design, but guessing it'd include some typography which it currently doesn't have (unlike the category). Could be a separate issue/feature though.

## Expected result  
Unless there's a reason for it, maybe we can use the same shade between light and dark for category chart and use the same shade for the pie labels as well

## Attachments  
The sample from #11662 can be reused

### PR DESCRIPTION
**Pull request by desig9stein**
Closes #11663 

### Additional information (check all that apply):
 - [ ] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Review by simeonoff — DISMISSED** [review]

**Comment by ChronosSF** [comment]
@desig9stein , @simeonoff - this has a 13.2 label but targets master. If it is something for 14.0.x please retarget it to 14.0.x . If it is for 14.1, you can leave it as is.

**Comment by sdimchevski** [comment]
@desig9stein I saw the changes for the colors that you made.
colors for the category charts that need to be updated:

title-text-color: grays 900 (h6 active)
subtitle-text-color: grays 700 (subtitle 2 inactive)

x-axis-label-text-color: grays 700 (body 2 inactive)
x-axis-title-text-color: grays 900 (body 2 active)
x-axis-tick-stroke: grays 500

y-axis-label-text-color: grays 700 (body 2 inactive)
y-axis-title-text-color: grays 900 (body 2 active)
y-axis-tick-stroke: grays 500

pie chart:
label-outer-color - grays 700

**Review by simeonoff — DISMISSED** [review]

**Comment by damyanpetev** [comment]
@desig9stein @sdimchevski 
Not entirely sure what the `y-axis-tick-stroke` is the intended style here, since that applies to the tickmarks extending from the y axis line if it was visible (x axis stroke is  null/transparent by default), as in these:
image
However, those are also length 0 by default, as in not visible. 
The actual grid lines are controlled by the y axis major stroke which seems to be `rgba(196,196,196,1)` coming from the chart.

Somewhat similar with the x axis - the tickmarks are visible by default so those are styled correctly, but the axis line is not specified - it's the xAxisStroke which defaults to `rgba(158,158,158,1)` thankfully (equivalent to gray 500) but not explicitly controlled by the theme. Not sure if we should explicitly set that as well.

**Comment by sdimchevski** [comment]
@desig9stein @damyanpetev 
The colors for Y axis stroke and X axis stroke should be grays 500, ticks - grays 500, gridilines - grays 300
Currently xAxisStroke is correct but it is not set by the theme. yAxisMajorStroke - horizontal grid lines, wrong, not set
For pie chart the theme can set --label-extent: 30;

File: projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_category-chart.scss [hunk 1]
```diff
@@ -133,7 +133,9 @@ $light-category-chart: (
 
     'x-axis-strip': null,
 
-    'x-axis-stroke': null,
+    'x-axis-stroke': (
+        color: ('grays', 300),
```
Comment (damyanpetev) [inline projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_category-chart.scss:137]: This should be 500 instead, right?

**Review by simeonoff — APPROVED** [review]

**Pull request merged**